src/ResourceManagement/Sql/SqlDatabaseImpl.cs

```diff
@@ -626,6 +650,27 @@ public SqlDatabaseImportRequestImpl ImportBacpac(IStorageAccount storageAccount,
                 .ImportFrom(storageAccount, containerName, fileName);
         }
 
+        ///GENMHASH:D07614EC964723EC7E563278F8FD0BE6:9903ECC9F8F5837D63A44DAA11F79C8E
+        public async Task<Microsoft.Azure.Management.Sql.Fluent.ISqlDatabaseUsageMetric> ListUsageMetricsAsync(CancellationToken cancellationToken = default(CancellationToken))
```

Comment:
ok, so it looks like the entire file is not implemented :)

Reply:
Fixed